### DISCUSSION TITLE
It looks like this IBAN is already in use.

### DISCUSSION THREAD

**Discussion by piotrcichosz** [firefly-iii/firefly-iii#7392] [Firefly III] [answered]
I created new savings account, typed iban, name, title, start balance
then wanted to edit and remove "include in net worth" checkbox - but I'm receiving error about iban already in use
Fact, there's such IBAN in revenue accounts (created with some initial import)

can I convert it somehow from revenue to assets or should I just empty iban for revenue account that's related to 1 imported transaction?

**Comment by JC5** [comment, marked as answer]
Just empty the revenue account. Did it not complain when you created the savings account?

**Reply by piotrcichosz** [reply]
No, it did not, so probably validation is different on creating than updating.
Emptied and update account, so that works for me

**Reply by JC5** [reply]
Thanks, I'll check that out!
